generic/tohil.c

```diff
@@ -122,6 +122,76 @@ tohil_UTF8ToTclDString(Tcl_Interp *interp, char *utf8String, int utf8StringLen,
     return Tcl_ExternalToUtfDString(utf8encoding, utf8String, utf8StringLen, ds);
 }
 
+//
+// tohil_UndentPython - remove consistent indenting from a python code block
+//
+static int tohil_UndentPython(Tcl_Interp *interp, char *string) {
+    char *indent = malloc(strlen(string)+1);
+    char *code_ptr = string;
+    char *indent_ptr = indent;
+    int seen_code = 0;
+
+    // look for indent
+    while (*code_ptr) {
+        // skip blank lines;
+        if(*code_ptr == '\n') {
```

Comment:
Do we need to consider '\r\n' sequences from Windows versions of Python?

Reply:
Thank you, I hadn't considered Windows.